web/src/pages/aqua_sign_wokflow/PdfWorkflowPage.tsx

```
@@ -29,7 +35,7 @@ export default function PdfWorkflowPage() {
 
             for (let i = 0; i < hashesToLoopPar.length; i += 3) {
                   const batch = hashesToLoopPar.slice(i, i + 3)
-                  //  console.log(`Processing batch ${i / 3 + 1}:`, batch)
+                  (`Processing batch ${i / 3 + 1}:`, batch)
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)